.ci/manywheel/build_cuda.sh

```diff
@@ -114,6 +115,14 @@ if [[ $USE_CUSPARSELT == "1" && $CUDA_VERSION == "11.8" ]]; then
         )
 fi
 
+
+# Turn USE_CUFILE off for CUDA 11.8
+# since nvidia-cufile-cu11 is not available in PYPI
+if [[ $USE_CUFILE == "1" && $CUDA_VERSION == "11.8" ]]; then
+    export USE_CUFILE=0
+fi
+
+
 if [[ $CUDA_VERSION == "12.4" || $CUDA_VERSION == "12.6" ]]; then
     export USE_STATIC_CUDNN=0
     # Try parallelizing nvcc as well
```

Comment:
I believe you also need to add cufile under ``if [[ -z "$PYTORCH_EXTRA_INSTALL_REQUIREMENTS" ]]; then`` if $USE_CUFILE is set